### PR TITLE
Add LE3BytesEnumField and LEX3BytesEnumField

### PR DESCRIPTION
**Pull request by antoniovazquezblanco**
**Checklist:**

-   [x] If you are new to Scapy: I have checked [CONTRIBUTING.md](https://github.com/secdev/scapy/blob/master/CONTRIBUTING.md) (esp. section submitting-pull-requests)
-   [x] I squashed commits belonging together
-   [x] I added unit tests or explained why they are not relevant
-   [x] I executed the regression tests (using `cd test && ./run_tests` or `tox`)
-   [x] If the PR is still not finished, please create a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/)

This PR adds two new field definitions. Those two new fields will be of use in future PRs to the bluetooth layer. Tests are added meanwhile.

Thanks

### CODE REVIEW

**Review by polybassa — APPROVED** [review]

File: test/fields.uts [hunk 1]
```diff
@@ -1199,6 +1199,286 @@ class Breakfast(Packet):
...
+
+assert f.i2repr_one(None, 0) == 'Foo'
+assert f.i2repr_one(None, 1) == 'Bar'
+assert f.i2repr_one(None, 2) == '0x2'
+assert f.i2repr_one(None, 0xff) == '0xff'
+
+assert f.i2repr_one(None, 0) == 'Foo'
+assert f.i2repr_one(None, 1) == 'Bar'
+assert f.i2repr_one(None, 2) == '0x2'
+assert f.i2repr_one(None, 0xff) == '0xff'
```
Comment (gpotter2) [inline test/fields.uts:1466]: What's with the duplicated tests?

Comment (antoniovazquezblanco) [reply test/fields.uts]: Solved. Thanks

**Review by polybassa — DISMISSED** [review]

**Review by gpotter2 — COMMENTED** [review]
So generally I don't think it's useful to have copy pasted all of EnumField tests, when it is reusing the same implementation behind the scenes.

Just 2 lines would have been enough to test it was called properly.

File: test/fields.uts [hunk 2]
```diff
@@ -1199,6 +1199,208 @@ class Breakfast(Packet):
...
+assert fcb.i2repr_one(None, 11) == repr(11)
+
+conf.noenum.remove(f, rf, lf, fcb)
+
+assert f.i2repr_one(None, RandNum(0, 10)) == '<RandNum>'
+assert rf.i2repr_one(None, RandNum(0, 10)) == '<RandNum>'
+assert lf.i2repr_one(None, RandNum(0, 10)) == '<RandNum>'
+assert fcb.i2repr_one(None, RandNum(0, 10)) == '<RandNum>'
+
+True
```
[the remark this answers is not in the document]
Comment (antoniovazquezblanco) [reply test/fields.uts:1239]: Removed

**Comment by antoniovazquezblanco** [comment]
I used EnumField tests as a starting point. I do not have that deep of a scapy knowledge and it was useful to perform all that testing just to ensure that what I programmed was working as expected and producing the desired result because I did not know what implications may have any small change in my implementation. While I am learning more and more about scapy I now have the knowledge to understand that many of this tests are not needed but I am still learning and may require some guidance.

I removed what I think may not be needed but may have overdone it... Thanks for the review and the patience! :)

**Review by gpotter2 — APPROVED** [review]

**Comment by antoniovazquezblanco** [comment]
Easier, shorter, better... Taking notes. I hope the next is less work for you. Thank you very much!

**Pull request merged**